src/services/files.js

```diff
@@ -74,4 +76,82 @@ module.exports = class FilesHelper {
 			throw error
 		}
 	}
+
+	static async getSignedUrlByOrgId(fileName, id, orgId, dynamicPath) {
+		try {
+			let destFilePath
+			if (dynamicPath != '') {
+				destFilePath = dynamicPath + '/' + fileName
+			} else {
+				destFilePath = `session/${id}-${new Date().getTime()}-${fileName}`
+			}
+			let response
+			if (process.env.CLOUD_STORAGE === 'GCP') {
+				response = await cloudServices.getGcpSignedUrl(destFilePath)
+			} else if (process.env.CLOUD_STORAGE === 'AWS') {
+				response = await cloudServices.getAwsSignedUrl(destFilePath)
+			} else if (process.env.CLOUD_STORAGE === 'AZURE') {
+				response = await cloudServices.getAzureSignedUrl(destFilePath)
+			} else if (process.env.CLOUD_STORAGE === 'OCI') {
+				response = await cloudServices.getOciSignedUrl(destFilePath)
+			}
+			const defaultOrgId = await getDefaultOrgId()
+			if (!defaultOrgId) {
+				return responses.failureResponse({
+					message: 'DEFAULT_ORG_ID_NOT_SET',
+					statusCode: httpStatusCode.bad_request,
+					responseCode: 'CLIENT_ERROR',
+				})
+			}
+
+			const data = { sample_csv_path: destFilePath }
+			if (orgId != defaultOrgId) {
+				await organisationExtensionQueries.update(data, orgId)
+			}
+
+			response.destFilePath = destFilePath
+			return responses.successResponse({
+				message: 'SIGNED_URL_GENERATED_SUCCESSFULLY',
+				statusCode: httpStatusCode.ok,
+				responseCode: 'OK',
+				result: response,
+			})
+		} catch (error) {
+			throw error
+		}
+	}
+
+	static async getDownloadableUrlByOrgId(orgId) {
```

Comment:
Why do we need this function, why aren't we reusing the existing downloadUrl()?

Reply:
@nevil-mathew This is needed to download csv based on orgId , if there is csv present for that orgId it will download the custom csv else the default csv